garage/tf/policies/categorical_mlp_policy.py

```diff
@@ -51,7 +51,7 @@ def __init__(
             with tf.name_scope(self._prob_network_name):
                 prob_network_outputs = L.get_output(prob_network.output_layer)
             self._f_prob = tensor_utils.compile_function(
-                [prob_network.input_layer.input_var], [prob_network_outputs])
+                [prob_network.input_layer.input_var], prob_network_outputs)
```

Comment:
This problem also appears in garage/tf/policies/deterministic_mlp_policy.py. Should I change it?

Reply:
Yes, please do that.